### ISSUE TITLE
minidump files are generated if you right-clicked a location in the navigation pane

### ISSUE THREAD

**Issue by wk-952** [x64dbg/x64dbg#459]
1) Debug any program and make any patches.
2) Press `Ctrl + P` --> Patch File --> select any location from the left navigation pane.
3) Right click on that location.
4) A minidump file will be generated in that location.

- Sometimes if you select the Desktop directly (without right-clicking), a minidump file will be
generated, but that's not consistent and once it happened you can't reproduce it again.
- No crashes, errors while patching or any other forms of failure occur,
it's just an annoying thing that happens, so i thought i might point it out (especially since
the minidump files are generated in the target location not in the debugger directory as usual).

__EDIT:__ it also happens if you tried to open an application from the debugger `( F3 )`,
so basically anything that involves the directory browser will have this issue.

**Comment by Nukem9** [comment]
Got any dump files to upload? This is specific to windows versions & what shell extensions are installed.
The exception itself is more than likely a false positive.

**Comment by wk-952** [comment]
http://www89.zippyshare.com/v/33vinua4/file.html

**Comment by Nukem9** [comment]
Fixed. Although C++ exceptions will be ignored as a result......might causes problems in the future.

**Comment by wk-952** [comment]
If that might cause problems in the future, then there's no need to ignore these exceptions.

If generating the minidump files in the debugger location won't cause any problems,
then that would be fair enough, if not then please feel free to close here.

Seriously, there's no need to to ignore these exceptions if that might cause any problems later,
as i said in the first post it's not a big deal nor a critical thing, but just annoying by time,
nothing more than that.

**Comment by wk-952** [comment]
This is the minidump of the rare case in which if you selected the Desktop directly
(without right-clicking), a minidump file will be generated.
__Note:__ This is reproduced with snapshot `snapshot_2015-12-16_11-58`.
http://www46.zippyshare.com/v/jA5FG81w/file.html

Please, if ignoring exceptions from this one too might cause problems in the future, just ignore it.

**Comment by Nukem9** [comment]
I'll have to check more with the exception handler then. I removed C++ EH filtering for the time being - it's specific to installed explorer shell extensions.

As a note:
`AddVectoredContinueHandler` is probably best, no idea if it truly works the way I want.
`SetUnhandledExceptionFilter` have proven to be unreliable in the past (i.e someone else uses this function and replaces our callback)
